src/questions/css-questions.md

```diff
@@ -35,3 +35,4 @@ permalink: /questions/css-questions/index.html
 * Can you explain the difference between coding a web site to be responsive versus using a mobile-first strategy?
 * Have you ever worked with retina graphics? If so, when and what techniques did you use?
 * Is there any reason you'd want to use `translate()` instead of *absolute positioning*, or vice-versa? And why?
+* Do you think it's necessary to follow CSS naming conventions? And why?
```

Comment:
what do you mean by saying `CSS naming conventions` ?

Reply:
The main idea behind CSS naming conventions is to make the CSS classes/selectors as informative and readable as possible. Naming conventions for classes/selectors are the major part of what a methodology like SMACSS, BEM, or OOCSS provides.

references: 
1. [BEM 101](https://css-tricks.com/bem-101/)
2. [Modular CSS naming conventions](http://thesassway.com/advanced/modular-css-naming-conventions)
3. [CSS Naming Conventions that Will Save You Hours of Debugging](https://medium.freecodecamp.org/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849)

Reply:
```Naming conventions for classes are the major part of what a methodology``` - from one of your articles. Maybe it worth to ask about methodologies ?

Reply:
just because naming itself doesn't help if you're not following other rules

Reply:
Well, these methodologies are basically to help/ease the naming conventions. It is not necessary that one needs to follow these methodologies, because one can have their own way of naming conventions. 

So my question "Do you think it's necessary to follow CSS naming conventions? And why?" would be more appropriate than asking "Do you follow any CSS naming methodologies?"

Reply:
actually, in this case for me it's just a rephrased version of `does it worth to follow convention on naming variables in code` ?

Reply:
okay, I get your point @vvscode , you want me to decline the PR or re-frame the question?

Reply:
@VishnuBaliga  I'm waiting for the opinion from other reviewers )

Reply:
Sorry. I'm here, I'm just slow on getting back to this project. There was also a lot of noise with  stalebot last month. 

I wouldn't ask about something like this because it usually doesn't matter. Most people are already in some sort of system set by someone else. I guess you could get an opinion about this, but there are a lot of things I'd ask about before I asked about CSS naming conventions. The one time it would be interesting would be at a very high level where you're trying to get at the "why" of the choices that people make. "I use _pattern x_ because I think that it helps developers... We implemented it as part our style guide and our configs at..." And even there it would be part of a larger question about standards and style, not something as pinpoint as this, 

So, as it stands. No. 

If you wanted to rephrase to be something that discusses named patterns and their pros and cons, then maybe?